integration/test/Testlib/HTTP.hs

```diff
@@ -101,6 +102,9 @@ onFailureAddResponse r m = App $ do
 
 data Versioned = Versioned | Unversioned | ExplicitVersion Int
 
+localRawBaseRequest :: HasCallStack => Service -> Versioned -> String -> App HTTP.Request
+localRawBaseRequest = rawBaseRequest OwnDomain
```

Comment:
What's the point of this definition? I'd rather see `rawBaseRequest OwnDomain` than `localRawBaseRequest` and have to look up what that is :shrug:.

Reply:
the idea was that somebody who doesn't know about `OwnDomain` can find this function and will understand that it's the thing they were looking for.  fewer questions from stupid developers you'll have to answer in the future.  :)

but no strong opinion, i'll remove it.  :+1: